### DISCUSSION TITLE
Why does the version start at v7.0.1 instead of v2 or v1 stable

### DISCUSSION THREAD

**Discussion by s3xybun** [vuejs/devtools#120] [General]
That was confused. When I use `pnpm update` today and it reported `v7.0.2 is available`, I thought that was a bug 😆 .

**Comment by webfansplz** [comment]
Because the [Legacy DevTools](https://github.com/vuejs/devtools/blob/aac74c0a4d3575692ef33bcf8b9f1a7161447150/packages/api/package.json#L3C2-L3C2) version is currently 6.5.1, we reused some of the previous package names after some discussion and consideration.

**Reply by s3xybun** [reply]
Oh, I am using `vite-plugin-vue-devtools` and the version of it is `1.0.0 rc`. `Legacy devtools` is a plugin of brower and `devtools-next && vite-plugin-vue-devtools`  is a packaging tool plugin, which caused a misunderstanding.

**Reply by webfansplz** [reply]
Yes, I can understand the confusion. We finally decided to unify the versions of all subpackages in order to better manage them.
